game/static/game/css/landing.css

```
@@ -333,15 +333,89 @@ body {
+    position: relative;
+    background: linear-gradient(
+        145deg,
+        rgba(255, 255, 255, 0.04),
+        rgba(255, 255, 255, 0.015)
+    );
+    border: 1px solid rgba(255, 255, 255, 0.08);
+    border-radius: 32px;
+    padding: 3.5rem;
+    overflow: hidden;
+
     display: flex;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)